sensor/kubernetes/listener/resources/secrets.go

```diff
@@ -290,6 +304,13 @@ func (s *secretDispatcher) processDockerConfigEvent(secret, oldSecret *v1.Secret
 				registryAddress, secret.GetNamespace(), secret.GetName())
 		}
 
+		if err := validateSecret(registryAddr, dce); err != nil {
+			log.Warnf("Not processing the registry with address %q found in secret %q from namespace %q: %v. ",
+				secret.GetName(), secret.GetNamespace(), registryAddr, err)
```

Comment:
Would it make sense to have information on skipped items in the administative events ?
This would be follow-up work

Reply:
Good point. I don't think so.